lib/components/home/artist_rails/artist_rail.js

```diff
@@ -29,12 +29,17 @@ class ArtistRail extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
-      artists: []
+      artists: [],
+      loadFailed: null,
```

Comment:
I’m thinking now that maybe this should just be `false` initially and you should also add this variable to the list of `state` types above (at line 27).